src/test/app/forms/models/amount.ts

```diff
@@ -114,12 +114,12 @@ describe('Amount', () => {
     it('should reject Nan Upper value', () => {
```

Comment:
I recon we should remove that test case as it is impossible to have NaN in current implementation